src/Card/Card.stories.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Card } from "./Card";
+import { cardAlignOptions } from "./Card.levers";
+import { Heading } from "../Heading/Heading";
+import { CtaLink } from "../CtaLink/CtaLink";
+
+export default {
+  title: "Simple/Card",
+  component: Card,
+  argTypes: {
+    align: {
+      control: {
+        type: "inline-radio",
+        options: cardAlignOptions,
+      },
+    },
+    isMinimal: {
+      control: {
+        type: "boolean",
+      },
+    },
+  },
+};
+
+const CardTemplate = ({ ...rest }) => (
+  <Card {...rest}>
+    {/* Developer would pass in their own card content components */}
```

Comment:
Hmm. Not entirely sure if I interpreted what we had envisioned for the Card wrapper 🤔  But here's my updated/super simplified take on it — assuming we want to give developers more flexibility with passing ino other components. Although I'm not sure if this is the best way to display this example within Storybook.

Thoughts, feedback, suggestions @sherakama @yvonnetangsu

Reply:
I only took a quick look but yes I believe you're going in the right direction. Looking great 🎉 ! If you could please move it under the "Composite" group (currently under "Simple") that'd be awesome 😄